### PR TITLE
Allow message to be sent when no ws connection

### PR DESCRIPTION
**Pull request by chadxz**
This patch adds the ability to send a message without needing to have
first connected via websocket.

### CODE REVIEW

**Comment by samuelg** [comment]
Ship it!

**Comment by ruffrey** [comment]
This is slick but a little weird. I like it. :+1:

File: lib/client.js [hunk 1]
```diff
@@ -820,6 +820,8 @@ function Client(options) {
      * @param {string} params.message required
      * @param {string} params.type='message' optional
      * @param {string} params.ccSelf=true optional
+     * @param {string} params.endpointId optional
+     * @param {function} callback
      * @auth endpoint, web socket
```
Comment (ruffrey) [inline lib/client.js]: we probably want to indicate that a web socket is not required

Comment (chadxz) [inline lib/client.js]: ah okay

Comment (samuelg) [inline lib/client.js]: Good catch.

Comment (chadxz) [inline lib/client.js]: I fixed it on master

**Pull request merged**